libtransmission/web.cc

```diff
@@ -332,6 +345,21 @@ class tr_web::Impl
         (void)curl_easy_setopt(e, CURLOPT_NOSIGNAL, 1L);
         (void)curl_easy_setopt(e, CURLOPT_PRIVATE, task);
 
+#if LIBCURL_VERSION_NUM <= CURL_VERSION_BITS(7, 76, 1)
```

Comment:
_nit_ would point to the version that includes the change, 7.77.0

```suggestion
#if LIBCURL_VERSION_NUM < CURL_VERSION_BITS(7, 77, 0)
// https://github.com/curl/curl/pull/6853
```

Reply:
Done, but I changed where this ifdef is located due to increased DNS traffic concern.